config/locales/client.en.yml

```diff
@@ -4,11 +4,18 @@ en:
       docs:
         help: "browse docs topics"
     docs:
+      or: "or"
```

Comment:
This doesn't usually translate well, I think you're better off using a single string and just including "categories or tags" in the string regardless of what is enabled in docs.